src/app/components/balance/balance-shimmer.tsx

```diff
@@ -0,0 +1,24 @@
+import { css } from 'leather-styles/css';
+import { Box } from 'leather-styles/jsx';
+
+export function BalanceShimmer({ ...rest }) {
```

Comment:
I think you only pass `width` and `height` as props, so we could just accept those instead of `..rest`?